lib/gollum-lib/wiki.rb

```diff
@@ -436,7 +421,7 @@ def update_page(page, name, format, data, commit = {})
       dir      = ::File.dirname(page.path)
       dir      = '' if dir == '.'
       filename = (rename = page.name != name) ?
-          Gollum::Page.cname(name) : page.filename_stripped
+          name : page.filename_stripped
```

Comment:
Right. However, as you mentioned, perhaps this PR should be restricted to only removing whitespace replacement. 😄